curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/authentication-strategies.english.md

```diff
@@ -8,11 +8,11 @@ forumTopicId: 301547
 ## Description
 <section id='description'>
 
-A strategy is a way of authenticating a user. You can use a strategy for allowing users to authenticate based on locally saved information (if you have them register first) or from a variety of providers such as Google or GitHub. For this project we will set up a local strategy. To see a list of the 100's of strategies, visit Passports site <a href='http://passportjs.org/' target='_blank'>here</a>.
+A strategy is a way of authenticating a user. You can use a strategy for allowing users to authenticate based on locally saved information (if you have them register first) or from a variety of providers such as Google or GitHub. For this project, we will set up a local strategy. To see a list of the hundreds of strategies, visit Passport's site <a href='http://passportjs.org/' target='_blank'>here</a>.
 
 Add <em>passport-local</em> as a dependency and add it to your server as follows: <code>const LocalStrategy = require('passport-local');</code>
 
-Now you will have to tell passport to <b>use</b> an instantiated LocalStrategy object with a few settings defined. Make sure this as well as everything from this point on is encapsulated in the database connection since it relies on it!
+Now you will have to tell passport to <b>use</b> an instantiated LocalStrategy object with a few settings defined. Make sure this (as well as everything from this point on) is encapsulated in the database connection since it relies on it!
```

Comment:
```suggestion
Now you will have to tell passport to <b>use</b> an instantiated <code>LocalStrategy</code> object with a few settings defined. Make sure this (as well as everything from this point on) is encapsulated in the database connection since it relies on it!
```